test/_detail_number.cpp

```diff
@@ -421,136 +452,139 @@ class number_test
-
-        fc("-1e308" );
-        fc("-1e-308" );
-        fc("-9999e300" );
-        fc("-999e100" );
-        fc("-99e10" );
-        fc("-9e1" );
-        fc( "9e1" );
-        fc( "99e10" );
-        fc( "999e100" );
-        fc( "9999e300" );
-        fc( "999999999999999999.0" );
-        fc( "999999999999999999999.0" );
-        fc( "999999999999999999999e5" );
-        fc( "999999999999999999999.0e5" );
-
-        fc( "0.00000000000000001" );
-
-        fc("-1e-1" );
-        fc("-1e0" );
-        fc("-1e1" );
-        fc( "0e0" );
-        fc( "1e0" );
-        fc( "1e10" );
-
-        fc( "0."
-            "00000000000000000000000000000000000000000000000000" // 50 zeroes
-            "1e50" );
-        fc( "-0."
-            "00000000000000000000000000000000000000000000000000" // 50 zeroes
-            "1e50" );
-
-        fc("0."
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000" // 500 zeroes
-            "1e600" );
-        fc("-0."
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000" // 500 zeroes
-            "1e600" );
 
-        fc( "0e"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000"
-            "00000000000000000000000000000000000000000000000000" // 500 zeroes
+        for (int i = 0 ; i < 2 ; ++i)
```

Comment:
FYI This is valid C++
```
for(auto b : {true, false})
```

Reply:
ah, nice.

Reply:
no longer needed. we're not parsing accurately atm